params/config.go

```diff
@@ -120,6 +161,8 @@ const (
 type ChainConfig struct {
 	ChainID *big.Int `json:"chainId"` // chainId identifies the current chain and is used for replay protection
 
+	Incompatible1Block *big.Int `json:"incompatible1Block"` // incompatible1 switch block (nil = no fork, 0 = already incompatible1)
```

Comment:
Doesn't it need `omitempty` tag?

Reply:
@yoomee1313 What do you think?

Reply:
yes, it needs omitempty tag. I'll fix it in later change(PR) which changes the name of this field! @aidan-kwon @KimKyungup

Reply:
Okay. Thanks!